common/storybook_utils.js

```diff
@@ -24,7 +24,7 @@ export const createTemplateFromVueFile = (args, argTypes, templateComponent) =>
  * @returns {string[]} icon component names
  */
 export function getIconNames () {
-  return Object.keys(dialtoneIcons).map(name => pascalCaseToKebabCase(name));
```

Comment:
Is that empty option necessary? I removed it before because I found that empty option at the beginning annoying.

Reply:
the problem is you have no way to set it to "no icon" without it

Reply:
Right, I know what you mean... Is it possible that you can add that option to only this story options instead of adding it to all of them?

I mean it's not a big problem, but I'll prefer to keep it the way it is now on the other components that uses this.

Reply:
I managed to get this to show (empty) instead of some weird empty string and the (empty) will be selected by default Are there still cases in storybook where we do NOT want this functionality?